pallets/runtime/src/bridge.rs

```diff
@@ -272,6 +291,50 @@ decl_module! {
             Ok(())
         }
 
+        /// Change the bridge limits.
```

Comment:
The subject of this sentence is "the function". Hence the verb should be "changes".